static/styles/dark_theme.css

```diff
@@ -526,7 +532,8 @@ body.dark-theme {
     .starred-messages-sidebar-menu-icon:hover,
     .drafts-sidebar-menu-icon:hover,
     .stream-sidebar-menu-icon:hover,
-    .topic-sidebar-menu-icon:hover {
+    .topic-sidebar-menu-icon:hover,
+    #show_all_private_messages {
```

Comment:
The feels like the wrong place to add this rule, given that every other element here is a hover element.

I'm also not sure why `!important` is required for the color of this element; generally we want a comment explaining any new `!important` usage.